torch_xla/core/xla_model.py

```diff
@@ -840,6 +840,15 @@ def mark_step(wait=False):
   torch_xla._XLAC._set_all_reduce_token(devctx.device, None)
 
 
+def get_xla_tensors_stablehlostep(tensors):
```

Comment:
we need doc strings to describe the behavior and input types, return types.

Reply:
Will add doc string for the updated API

Reply:
Perhaps these doc strings could also describe anticipated use cases?

Reply:
Good idea, anticipated use cases are added.